.drone.yml

```diff
@@ -1,129 +1,130 @@
+    secrets:
+      - clamav_repo_token
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - docker build -t clamav .
+      - docker login -u="ukhomeofficedigital+drone" -p=$${CLAMAV_REPO_TOKEN} quay.io
+      - docker tag clamav:$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/clamav:$${DRONE_TAG}
+      - docker push quay.io/ukhomeofficedigital/clamav:$${DRONE_TAG}
     when:
       event: tag
   
-  build_clamav-rest:
-    image: docker:1.11
-    environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
-    commands:
-      - docker build -t clamav-rest clamav-rest
-    when:
-      event: tag  
-  clamav_to_quay:
-    image: docker:1.11
+  clamav-rest_to_quay:
+    image: docker:17.09.0-ce
+    secrets:
+      - clamav_repo_token
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - docker login -u="ukhomeofficedigital+drone" -p=${CLAMAV_REPO_TOKEN} quay.io
-      - docker tag clamav quay.io/ukhomeofficedigital/clamav:${DRONE_COMMIT_SHA}
-      - docker push quay.io/ukhomeofficedigital/clamav:${DRONE_COMMIT_SHA}
+      - docker login -u="ukhomeofficedigital+drone" -p=$${CLAMAV_REPO_TOKEN} quay.io
+      - docker tag clamav-rest:$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/clamav-rest:$${DRONE_COMMIT_SHA}
+      - docker tag clamav-rest:$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/clamav-rest:latest
+      - docker push quay.io/ukhomeofficedigital/clamav-rest:$${DRONE_COMMIT_SHA}
+      - docker push quay.io/ukhomeofficedigital/clamav-rest:latest
     when:
-      event: tag
+      branch: master
+      event: push
   
-  clamav-rest_to_quay:
-    image: docker:1.11
+  tagged_clamav-rest_to_quay:
+    image: docker:17.09.0-ce
+    secrets:
+      - clamav_repo_token
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - docker login -u="ukhomeofficedigital+drone" -p=${CLAMAV_REPO_TOKEN} quay.io
-      - docker tag clamav-rest quay.io/ukhomeofficedigital/clamav-rest:${DRONE_COMMIT_SHA}
-      - docker push quay.io/ukhomeofficedigital/clamav-rest:${DRONE_COMMIT_SHA}
+      - docker login -u="ukhomeofficedigital+drone" -p=$${CLAMAV_REPO_TOKEN} quay.io
+      - docker tag clamav-rest:$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/clamav-rest:$${DRONE_TAG}
+      - docker push quay.io/ukhomeofficedigital/clamav-rest:$${DRONE_TAG}
     when:
       event: tag
   
   trigger_deploy_to_platform-services_dev:
     image: quay.io/ukhomeofficedigital/kd:v0.2.2
+    secrets:
+      - kube_token_dev
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - kd --insecure-skip-tls-verify --namespace=virus-scan --kube-server=https://kube-dev.dsp.notprod.homeoffice.gov.uk --retries 50 --kube-token=${KUBE_TOKEN_DEV} --file k8s/clamd-api.yaml
```

Comment:
new drone isnt peered with, dsp notprod or prod.

Reply:
new drone is peers with DSP CI, NOTPROD and PROD :-)

Reply:
I'll change it back then...